torch/_functorch/aot_autograd.py

```diff
@@ -496,7 +506,12 @@ def gen_alias_from_base(aliased_base_tensor, target_meta_tensor, target_requires
         else:
             reshaped_base_tensor = aliased_base_tensor
         out = target_meta_tensor._view_func(reshaped_base_tensor)
-        if out is not None:
+        # This shape mismatch can happen due to a bug in inplace/view handling in autograd.
+        # Try putting a breakpoint here and running `test/functorch/test_aotdispatch TestAOTAutograd`
+        # To see where that happens.
+        # Also, https://github.com/pytorch/pytorch/issues/49825
```

Comment:
Tbh, the issue isn't terribly clear about what the problem is. An update would be helpful. Cc @albanD

Reply:
Yep, now that we have the context manager for turning on view replay this is easier. I added a repro in https://github.com/pytorch/pytorch/issues/49825#issuecomment-1422865300.